test/service_tests/image_service_test.dart

```diff
@@ -80,7 +81,7 @@ void main() {
       //using this asset as the test asset
       final file = File('assets/images/Group 8948.png');
       final fileString = await imageService.convertToBase64(file);
-      print(fileString);
+      expect(fileString, isA<String>());
```

Comment:
Not merely any string, but should be the expected base64 string.

Reply:
@literalEval  this time instead of comparing the strings. i have compared the bytes generated by the `actual file Object` and the bytes generated by the `decoded string`.